modules/cudf/src/series/bool.ts

```diff
@@ -28,4 +30,97 @@ export class Bool8Series extends NumericSeries<Bool8> {
   get data() {
     return new Uint8ClampedBuffer(this._col.data).subarray(this.offset, this.offset + this.length);
   }
+
+  _prepare_scan_series(skipna: boolean) {
```

Comment:
```suggestion
protected _prepare_scan_series(skipna: boolean) {
```

(anticipating this one)